modules/log-upgrade/6x_deploying-a-clusterlogforwarder-observability-custom-resource.adoc

```diff
@@ -0,0 +1,66 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-05-20
+:_mod-docs-content-type: PROCEDURE
+
+[id="deploying-a-clusterlogforwarder-observability-custom-resource_{context}"]
+= Deploying a ClusterLogForwarder observability custom resource
+
+Deploying a ClusterLogForwarder observability custom resource (CR) by using the {oc-first} command.
+
+.Prerequisites
+* You have administrator permissions.
+* You installed the {oc-first}.
+
+.Procedure
+. Create a ClusterLogForwarder observability custom resource (CR).
++
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: collector
+  namespace: openshift-logging
+spec:
+  serviceAccount:
+    name: collector
```

Comment:
In the steps is documented to create the serviceAcount "logging-collector" and here, it's set "collector", then, this won't work.

If it's used the legacy serviceAccount, it should be "logcollector", if it's created a new serviceAccount, the name set in the steps here is "logging-collector"

Reply:
changed to <Name_of_service_account>